cli/network.go

```diff
@@ -0,0 +1,162 @@
+var networkDescription = "Manager the networks in pouchd. " +
+	"It contains the functions of create/remove/list/inspect network, 'driver' is used to list drivers that pouch support. " +
+	"Now bridge network is supported in pouchd defaulted, it will be initialized when pouchd starting."
+
+// NetworkCommand is used to implement 'network' command.
+type NetworkCommand struct {
+	baseCommand
+}
+
+// Init initializes NetworkCommand command.
+func (n *NetworkCommand) Init(c *Cli) {
+	n.cli = c
+
+	n.cmd = &cobra.Command{
+		Use:   "network [command]",
+		Short: "Manage pouch networks",
+		Long:  networkDescription,
+		Args:  cobra.MinimumNArgs(1),
+	}
+
+	c.AddCommand(n, &NetworkCreateCommand{})
+}
+
+// networkCreateDescription defines the comand network create description.
+var networkCreateDescription = "Create a network in pouchd. " +
+	"It must specify network's name and driver. You can use 'network driver' to get drivers that pouch support."
+
+// NetworkCreateCommand is used to implement 'network create' command.
+type NetworkCreateCommand struct {
+	baseCommand
+
+	name       string
+	driver     string
+	gateway    string
+	ipRange    string
+	ipamDriver string
+	ipamOpts   []string
+	subnet     []string
+	options    []string
+	labels     []string
+}
+
+// Init initializes NetworkCreateCommand command.
+func (n *NetworkCreateCommand) Init(c *Cli) {
+	n.cli = c
+
+	n.cmd = &cobra.Command{
+		Use:   "create <NAME>",
+		Short: "Create a pouch network",
+		Long:  networkCreateDescription,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return n.runNetworkCreate(args)
+		},
+		Example: networkCreateExample(),
+	}
+
+	n.addFlags()
+}
+
+// addFlags adds flags for specific command.
+func (n *NetworkCreateCommand) addFlags() {
+	flagSet := n.cmd.Flags()
+
+	flagSet.StringVarP(
+		&n.name,
+		"name",
+		"n",
+		"",
+		"the name of network")
+	flagSet.StringVarP(
+		&n.driver,
+		"driver",
+		"d",
+		"bridge",
+		"the driver of network")
+	flagSet.StringVar(
+		&n.gateway,
+		"gateway",
+		"",
+		"the gateway of network")
+	flagSet.StringVar(
+		&n.ipRange,
+		"ip-range",
+		"",
+		"the range of network's ip")
+	flagSet.StringSliceVar(
+		&n.subnet,
+		"subnet",
+		nil,
+		"the subnet of network")
+	flagSet.StringVar(
+		&n.ipamDriver,
+		"ipam-driver",
+		"",
+		"the ipam driver of network")
+	flagSet.StringSliceVarP(
+		&n.options,
+		"option",
+		"o",
+		nil,
+		"create network with options")
+	flagSet.StringSliceVarP(
+		&n.labels,
+		"label",
+		"l",
+		nil,
+		"create network with labels")
+}
+
+// runNetworkCreate is the entry of NetworkCreateCommand command.
+func (n *NetworkCreateCommand) runNetworkCreate(args []string) error {
+	name := n.name
+	if len(args) != 0 {
+		name = args[0]
+	}
+	if name == "" {
+		return fmt.Errorf("need to specified network's name")
+	}
+
+	networkRequest := &types.NetworkCreateRequest{
+		Name: name,
+	}
+
+	if n.driver == "" {
+		return fmt.Errorf("need to specified network's driver")
```

Comment:
network driver cannot be empty?

Reply:
yes, although the default network driver is bridge, we recommend user can support the network driver and its configurations.

Reply:
I mean we can use `network driver cannot be empty` to replace `need to specified network's driver`. 
The same above. 😄

Reply:
oh, sorry, Misunderstanding